src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("core.basesyntax.Bulldozer started work");

```

Comment:
```suggestion
        System.out.println("Bulldozer started work");

```

same for other classes